ke/compiler.py

```diff
@@ -259,14 +254,21 @@ def compile_operator(o, macros):
         else:
             min = int(min)
             max = int(max)
-        return asm.Multiple(min, max, True, sub)
+        
+        return asm.Multiple(min, max, get_greediness_by_name_of_operator(o.name), sub)
```

Comment:
any other name should be an error (e.g. `[1-3:aur "hi"]` should be an error)

Reply:
How can we possibly approach this? 'fewest' is not really an operator, but rather a 'cheap-haxed' way that we check if the incoming operator matches and then return either true or false. so 'aur' would compile but doesn't set the flag appropriately. There are many more operators that access this check so throwing an error breaks it down.
I might be wrong tho.